public/app/features/alerting/unified/components/rule-viewer/AlertRuleMenu.tsx

```diff
@@ -131,8 +146,23 @@ const AlertRuleMenu = ({
     (rulerRuleType.grafana.rule(rulerRule) && isPausedRule(rulerRule)) ||
     (prometheusRuleType.grafana.rule(promRule) && promRule.isPaused);
 
+  // todo: make this new menu item for enrichments an extension of the alertrulemenu items. For first iteration, we'll keep it here.
+  const canManageEnrichments =
+    canEditRule &&
+    ruleUid &&
+    handleManageEnrichments &&
+    config.featureToggles.alertingEnrichmentPerRule &&
+    config.featureToggles.alertEnrichment;
+
   const menuItems = (
     <>
+      {canManageEnrichments && (
+        <Menu.Item
+          label={t('alerting.alert-menu.manage-enrichments', 'Manage enrichments')}
+          icon="edit"
+          onClick={handleManageEnrichments}
+        />
+      )}
```

Comment:
I think, it would be better if this were handled by the enterprise code. This could be set up as an extension point for custom actions, where the enterprise code could then hook in.

In the current implementation, does the current OSS code display the drawer if the necessary feature toggles are enabled?